### ISSUE TITLE
Unable to reproduce fb15k results from downstream tasks

### ISSUE THREAD

**Issue by Npasic1** [facebookresearch/PyTorch-BigGraph#81]
I have ran the command
`torchbiggraph_example_fb15k`
in order to try and reproduce "Predicting the score of an edge" and "Ranking" result from this link https://torchbiggraph.readthedocs.io/en/latest/downstream_tasks.html#using-the-embeddings
However, for the edge score I get this result:
`tensor([[21.2358]], grad_fn=<SumBackward2>)`
I expected a number between 0 and 1 and close to 1 (because this statement is true). 
The scores do make sense because for Berlin being a capital of France I get 14.6, and for Barack Obama being a capital of France I get score 3.25.
I assume some sort of normalization function needs to be applied in order to normalize these scores?

For the Ranking example, I get 5 results:
`['/m/0f8l9c', '/m/05qtj', '/m/01c6rd', '/m/09b83', '/m/0p0mx']`
The first one is France, and the second one is Paris -- I didn't expect to see France here since we are determining the capital of France.

Not sure if I'm missing something here, I used the scripts from the link without changing them.

Thanks!

**Comment by lerks** [comment]
1. The score of an edge isn't necessarily a value between 0 and 1, it can be any real number. It's not a probability, it's a score. There are several ways of converting it to a probability, if needed: passing it through a sigmoid, doing a softmax, .... Which one is the best suited depends on the loss function you used for training (for FB15k that's the softmax).

2. Maybe you're expecting too much from the embeddings: there will always be some noise, because in addition to the "is capital of" edges there are a ton of other edges pulling things close together or far apart. The fact that Paris ranks second is already a pretty good result. Additional training may fine-tune some of these instances but you will never get to a point where they all work perfectly (that would mean having a MRR of 1, which is not our case).

**Comment by Npasic1** [comment]
Thank you for the quick response! That makes sense.
Regarding 2. - definitely great results, I was just wondering if maybe I was doing something wrong since the docs listed 5 French cities and not France :)
